bindings/cs/MongoDB.Libmongocrypt/BinarySafeHandle.cs

```diff
@@ -33,6 +33,15 @@ private BinarySafeHandle()
         {
         }
 
+        private BinarySafeHandle(IntPtr ptr)
+            : base(ptr, false)
+        {
+        }
+
+        public static BinarySafeHandle FromIntPtr(IntPtr ptr)
```

Comment:
I don't see this method usages in the repo. Is it involved in native library functioning?